wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1296,6 +1297,56 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag(
+    "wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True
+)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    modifier = "⌘" if is_mac else "Ctrl"
+    shortcuts = {
+        ("common-actions", _("Common Actions")): [
+            {"label": _("Copy"), "shortcut": f"{modifier} + c"},
+            {"label": _("Cut"), "shortcut": f"{modifier} + x"},
+            {"label": _("Paste"), "shortcut": f"{modifier} + v"},
+            {
+                "label": _("Paste without formatting"),
+                "shortcut": f"{modifier} + shift + v",
+            },
+            {"label": _("Undo"), "shortcut": f"{modifier} + z"},
+            {"label": _("Redo"), "shortcut": f"{modifier} + y"},
+        ],
+        ("page-editing-actions", _("Page editing actions")): [
+            {"label": _("Save"), "shortcut": f"{modifier} + s"},
```

Comment:
```suggestion
            {"label": _("Save (draft)"), "shortcut": f"{modifier} + s"},
```

Reply:
This is really a nitpick. For the first iteration we can go with "Save".

Just to explain myself: "Save changes"  or "Save" do not clarify that pages don't have a "Save" option. Pages have a workflow with "Save draft" and "Publish". Therefore, "Save" on a page will always be result in a "Save draft".  I think some users expect saving a already publish page will publish their changes. But that won't happen. They just get a new draft.

The modal with shortcuts is like a cheatsheet. And it is common for cheatsheets to have additional explanation. Maybe help texts? "A pages will save a draft." 

Nice to have. Not for now.

Reply:
Yep. Let's keep this simple for now, I anticipate we would have a way for the label/shortcut to be registered somehow at a View or Form level. 

Or we would drive this from the JS side and pull out the label from the element itself. 

As this is static for this first version I think we'll have to compromise with something generic. 

But if you have ideas on how we could provide a way for this to be registered - I'm super keen to hear them. Add a comment to https://github.com/wagtail/wagtail/issues/3949

Reply:
Done